### DISCUSSION TITLE
MMX version 1.4

### DISCUSSION THREAD

**Discussion by luisdev** [moneymanagerex/moneymanagerex#4069] [General]
I see that there is no 1.4 version of MMX on the https://github.com/moneymanagerex/moneymanagerex/releases?page=2 page. It jumps from v1.3.6 to v1.5.0. What happened to MMX v1.4? Was it never released?

I'm just wondering... :smiley:   I'm not trying to install it...

**Comment by tactilis** [comment]
See the comment from @vomikan here:

https://github.com/moneymanagerex/moneymanagerex/issues/2353#issuecomment-607730547

**Reply by luisdev** [reply]
Thanks :-)
